src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    protected String machine;
+
+    public Machine(String machine) {
+        this.machine = machine;
+    }

```

Comment:
The constructor parameter 'machine' should be renamed for clarity. If this is intended to represent the type or name of the machine, consider renaming the parameter to 'machineType' or 'machineName' to match the suggested field name change.